### PR TITLE
For #3443: Convert TextPercentageSeekBarPreference to Kotlin style

### PR DESCRIPTION
**Pull request by miDeb**
I simplified the code and removed some unused functionality as well, but it should not modify existing functionality. Maybe I sould add a test, though?
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by gabrielluong** [comment]
@NotWoods would you have a chance to review this? I realize I can't set you as a reviewer because of the org permissions, but I think you have the most valuable insight for this issue. Thank!

**Review by NotWoods — CHANGES_REQUESTED** [review]
This looks pretty good to me but it's a bit hard to follow the PR since there's a mix of style and functional changes. Could you split the changes into two commits: one just changing names and style (ie using let instead of if), and a second to remove unused functionality?

Everything past the line where `updateLabelValue` is declared looks good, prior to that its hard to follow the logic changes.

**Comment by gabrielluong** [comment]
@miDeb Thank you for the contribution! Let us know if you will have any opportunity to make the requested changes.

**Comment by Mugurell** [comment]
@miDeb The changes look good,
It's only a little bit of cleanup still needed as per https://github.com/mozilla-mobile/fenix/pull/14395#pullrequestreview-507103429 and there are also some conflicts that now need to be resolved.
Will you have time to implement these? If you need any help please let us know.

**Comment by miDeb** [comment]
I might have time to make the changes in the next week(s). And if there's anything I need help with, I will surely ask.

**Comment by Mugurell** [comment]
@miDeb I know a lot of time passed since. Are you still interested in refactoring this?

**Comment by gabrielluong** [comment]
🚧 **Pull Request Maintenance** 🚧 Thank for your contribution! We're closing this pull request since it has been stale for awhile. If the pull request is still relevant, please feel free to reopen or comment on any open issue if you're interested in contributing further. Thanks again! 🙇
